### PR TITLE
Edited the English README.md for correctness and clarity

### PR DESCRIPTION
**Pull request by vickychenglau**
I proofread your English README.md file and corrected all the grammar errors and awkward wording I found.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
 # What's JSBox?
 
-JSBox is actually an Integrated Development Environment for JavaScript, you can learn to code JavaScript here, we provided tons of utilities to enhance the develop experience.
+JSBox is an Integrated Development Environment for JavaScript. You can learn to code JavaScript here, as we provide tons of utilities to enhance the development experience.
 
-JSBox is amazing for both greenhands and experienced engineers, don't you wanna try?
+JSBox is amazing for both beginners and experienced engineers. Don't you wanna try it out?
 
-Don't be hesitated to create cool stuff, do it now
+Don't hesitate to create cool stuff—do it now!
```
Comment (cyanzhong) [inline README.md]: What does stuff-do mean?

File: README.md [hunk 2]
```diff
@@ -30,32 +30,32 @@ With these examples you can have a brief understanding for what can we do with J
...
 
-Of course you can write JavaScript, but the most important is you can create native iOS utilities, such as:
+Of course you can write JavaScript, but the most important thing is that you can create native iOS utilities, such as:
 
-- Upload photo you have just taken to the server
+- Upload photos you have just taken to the server
 - Create a widget to check bus information
 - Schedule tasks by leveraging date-based or location-based notifications
-- Design an extension for Photos app to search photos directly
+- Design an extension for a Photos app to search photos directly
```
Comment (cyanzhong) [inline README.md]: Photos app is a specific app I think, it is "the" Photos app.

**Comment by vickychenglau** [comment]
The character — is an em dash. It acts differently from a hyphen (-) and works like a semicolon (;) but keeps the phrases more closely connected than a semicolon. 
Basically, I'm not using stuff-do as a single term, but instead connecting the phrases "Don't hesitate to create cool stuff" and "do it now". Feel free to change it if you dislike that formatting or find it confusing.

As for the photo correction, I believe editing it yourself after accepting the merge will be more efficient. I assumed that it meant any photo app, so that was my mistake. My bad.

**Comment by cyanzhong** [comment]
Oh it's OK, I just don't have enough background to understand this usage, thanks for the knowledge you've provided.

**Comment by cyanzhong** [comment]
I'm heading to merge this one, thanks for your corrections again. 👍

**Pull request merged**